Userland/Applications/FontEditor/main.cpp

```diff
@@ -57,5 +54,16 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
 
     window->show();
 
+    auto path_to_load = path.is_empty() ? "/res/fonts/KaticaRegular10.font"sv : path;
+    auto file = TRY(FileSystemAccessClient::Client::the().request_file_read_only_approved(window, path_to_load));
+
+    if (!path.is_empty()) {
+        TRY(font_editor->open_file(file.filename(), file.release_stream()));
+    } else {
+        auto mapped_file = TRY(Core::MappedFile::map_from_file(file.release_stream(), path_to_load));
+        auto mutable_font = TRY(TRY(Gfx::BitmapFont::try_load_from_mapped_file(mapped_file))->unmasked_character_set());
+        TRY(font_editor->initialize({}, move(mutable_font)));
+    }
+
```

Comment:
can the if/else be reduced to simply `TRY(font_editor->open_file(file.filename(), file.release_stream()));`?

Reply:
Don't think so, we want to make a 'mutable copy' of Katica, we don't want the editor to open Katica by default and allow editing of it. (At least that's what the existing behavior is)

Reply:
Yeah, ignore this one. Both make mutable copies but we'd get permission denied (because of the initialized path) on a simple save which isn't very nice for a default.